issues 8 and 9 fixed

emestanza3 opened the pull request:
Made the side bar disappear while simulation is running and reappear when simulation isn't. When play button is pressed it hides it, the pause button re shows it. Also animation controller is above the upload button

emestanza3 committed:
issues 8 and 9 fixed

jbonzo reviewed (commented):
I can't seem to get this to work. I'm loading a trace, then clicking the play button. should that make the sidebar disappear. Also the animation is still below the upload.

emestanza3 committed:
updates so side bar issues

Check and see if the side bar dissapears now. Process to make this work should only be to click the play button, to make it reappear click the pause button

emestanza3 committed:
Update to cacheDetails.html

emestanza3 committed:
Update cacheDisplay.html

emestanza3 committed:
Update cacheDisplay.js

emestanza3 committed:
Delete cacheDisplay.js

emestanza3 committed:
Delete cacheDetail.html

emestanza3 committed:
Delete cacheDetail.js

emestanza3 committed:
Delete cacheDisplay.html

emestanza3 committed:
Update cacheDisplay.html

emestanza3 committed:
Merge branch 'master' into sidebar-issues
